### PR TITLE
Spelling and Grammar Fixes

### PR DESCRIPTION
**Pull request by klorpa**
## About The Pull Request
Fixes several errors to spelling, grammar, and punctuation.
## Changelog
:cl:
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/martian.dm [hunk 1]
```diff
@@ -520,7 +520,7 @@
 	crafting_complexity = FOOD_COMPLEXITY_3
 
 /obj/item/food/takoyaki/russian
-	name = "russian takoyaki"
+	name = "Russian takoyaki"
```
Comment (MrMelbert) [inline code/game/objects/items/food/martian.dm:523]: Might need to be `\improper`

**Comment by EricZilla** [comment]
A "few" and its 66, we salute you for your hard work

File: code/game/objects/items/food/martian.dm (hunk 1 above)
Comment (TheVekter) [reply code/game/objects/items/food/martian.dm:523]: You might be correct but isn't Russian a proper noun?

Comment (TheVekter) [reply code/game/objects/items/food/martian.dm:523]: Yup, makes sense. Thanks.

**Pull request merged**